src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -736,6 +736,14 @@ ssize_t sockinfo_tcp::tx(const tx_call_t call_type, const iovec* p_iov, const ss
 		return -1;
 	}
 
+#ifdef DEFINED_TCP_TX_WND_AVAILABILITY
+	if (!is_window_available(&m_pcb, p_iov[0].iov_len)) {
```

Comment:
@DanielLibenson I think that this check does not cover user passed data in iov size greater than 1.

Reply:
You are absolutely right, window availability feature will not support iov. 
I will add note/log message.